ClassLibrary1/ClassLibrary1/Class1.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ClassLibrary1
+{
+    public class Class1
```

Comment:
If you were submitting this branch for review in a production environment, you'd want to make sure it doesn't contain unneeded files like this one. I make a practice of always reviewing my branch changes to ensure I don't need to clean anything up like this before review.